.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: python
+python:
+- "3.7"
+-
+install:
+#install conda
+ - sudo apt update
+ - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+ - bash miniconda.sh -b -p $HOME/miniconda
+ - source "$HOME/miniconda/etc/profile.d/conda.sh"
+ - hash -r
+ - conda config --set always_yes yes --set changeps1 no
+ - conda update -q conda
+
+ - conda info -a
+
+ - conda create --name pd
+ - conda activate pd
+ - conda install tensorflow jupyterlab pandas regex matplotlib cython grpcio-tools pytorch scikit-learn
```

Comment:
Can you move this list of packages and the one on the next line to an `environment.yml` file, and update `env.sh` to read package names the same `environment.yml` file?

Reply:
Should be fixed by my latest commit